### PR TITLE
Run docker as nonroot

### PR DESCRIPTION
**Pull request by patriotyk**

### CODE REVIEW

File: Dockerfile.nop11 [hunk 1]
```diff
@@ -54,5 +54,8 @@ RUN apt-get update && apt-get -y install \
   valgrind \
   wget
 
+RUN useradd testuser
 WORKDIR aktualizr
-ADD . src
+RUN chown -R testuser:testuser .
+ADD --chown=testuser:testuser . src
+USER testuser:testuser
```
Comment (patrickvacek) [inline Dockerfile.nop11]: Why is the nop11 version different from the others? It looks like the RUN and ADD commands are doing the same thing to the current source directory, but I don't know enough about what's really happening to know. If that logic really is necessary, could you leave a comment in the Dockerfile to explain it?

Comment (patriotyk) [reply Dockerfile.nop11]: It is not easy to run coverage docker to run tests with user permissions because softhsm requires root access.

Comment (patrickvacek) [reply Dockerfile.nop11]: I believe that, but isn't the nop11 Dockerfile the one that explicitly *doesn't* use softhsm?

Comment (patriotyk) [reply Dockerfile.nop11]: Yes, it is. But why do You ask?

Comment (patrickvacek) [reply Dockerfile.nop11]: If nop11 doesn't use softhsm, why is it the case that it needs the special handling for softhsm? I would've expected it to be the other way around.

**Pull request merged**